src/hotspot/share/oops/compressedOops.hpp

```diff
@@ -34,23 +34,18 @@
 class outputStream;
 class ReservedHeapSpace;
 
-struct NarrowPtrStruct {
```

Comment:
Not an objection, just a complaint on how the GitHub diff view makes this confusing to review